keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -292,14 +329,94 @@ Checkpointing containers will be possible again.
 
 ###### Are there any tests for feature enablement/disablement?
 
-Currently no.
+Currently the test will automatically be skipped if the feature is not enabled.
```

Comment:
Ah, okay, that sounds easy to implement. This basically means the test needs to be less clever and instead of skipping the test it should `PASS` if the feature is turned off and does not work. Will add something here.

Reply:
Let's document the tests we want to add so we don't forget during coding time.

Reply:
Good idea. I documented the tests which will be introduced.